### PR TITLE
(DOCSP-27006): Multi-user applications

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

@realm/reactify Multi-user Applications page. 

Also added some missing content that's in other SDKs.

### Jira

- https://jira.mongodb.org/browse/DOCSP-27006

### Staged Changes

- [Multi-User Applications (React Native)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27006/sdk/react-native/manage-users/multi-user-applications/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/react-native/manage-users/multi-user-applications.txt [hunk 1]
```diff
@@ -10,116 +10,112 @@ Multi-User Applications - React Native SDK
...
 
+The Realm SDK allows multiple :ref:`users <user-accounts>` to be logged in to an
+app simultaneously on a given device. Atlas App Services client applications run
+in the context of a single active user even if multiple users are logged in
+simultaneously. You can quickly switch between authenticated users without
+requiring them to log in again.
+
+.. important::
+
+   **Any logged in user may become the active user without re-authenticating.**
```
Comment (cbush) [inline source/sdk/react-native/manage-users/multi-user-applications.txt:21]:
```suggestion
   **Any logged-in user may become the active user without re-authenticating.**
```

File: source/sdk/react-native/manage-users/multi-user-applications.txt [hunk 2]
```diff
@@ -10,116 +10,112 @@ Multi-User Applications - React Native SDK
...
+   providing their credentials or requiring their explicit permission. If your
+   application requires stricter authentication, avoid switching between users
+   and prefer to explicitly log the active user out before authenticating
+   another user.
+
+User Account States
+-------------------
+
+When a user first logs in through a Realm SDK on a given device or
+browser, the SDK saves the user's information and keeps track of the user's
```
Comment (cbush) [inline source/sdk/react-native/manage-users/multi-user-applications.txt:33]: "Browser" seems inapplicable to the RN SDK?

**Review by cbush — APPROVED** [review]
Lööks gööd

File: source/sdk/react-native/manage-users/multi-user-applications.txt (hunk 2 above)
Comment (mongodben) [reply source/sdk/react-native/manage-users/multi-user-applications.txt:33]: yeah definitely. will update. not sure how that got in there.

Comment (mongodben) [inline source/sdk/react-native/manage-users/multi-user-applications.txt:33]:
```suggestion
When a user first logs in through a Realm SDK on a device,
the SDK saves the user's information and keeps track of the user's
```

**Pull request merged**